app/scripts/models/auth_brokers/fx-ios-v1.js

```diff
@@ -11,13 +11,88 @@ define(function (require, exports, module) {
   'use strict';
 
   const _ = require('underscore');
-  const FxDesktopV1AuthenticationBroker = require('../auth_brokers/fx-desktop-v1');
+  const FxDesktopChannel = require('../../lib/channels/fx-desktop-v1');
+  const FxSyncChannelAuthenticationBroker = require('../auth_brokers/fx-sync-channel');
+  const HaltBehavior = require('../../views/behaviors/halt');
   const NavigateBehavior = require('../../views/behaviors/navigate');
+  const Url = require('../../lib/url');
   const UserAgent = require('../../lib/user-agent');
 
-  const proto = FxDesktopV1AuthenticationBroker.prototype;
+  const proto = FxSyncChannelAuthenticationBroker.prototype;
+
+  const FxiOSV1AuthenticationBroker = FxSyncChannelAuthenticationBroker.extend({
+    type: 'fx-desktop-v1',
+
+    commands: {
+      CAN_LINK_ACCOUNT: 'can_link_account',
+      CHANGE_PASSWORD: 'change_password',
+      DELETE_ACCOUNT: 'delete_account',
+      LOADED: 'loaded',
+      LOGIN: 'login'
+    },
+
+    defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterForceAuth: new HaltBehavior(),
+      // about:accounts displays its own screen after password reset, no
+      // need to show anything.
+      afterResetPasswordConfirmationPoll: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterSignIn: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
```

Comment:
In the [initialize method](https://github.com/mozilla/fxa-content-server/pull/7037/files#diff-110efcbbe9c2689e314cc3faa97baf8bL40) we override both `afterSignInConfirmationPoll` and `afterSignUpConfirmationPoll`. Since we no longer support fx-desktop-v1, we might as well make those the defaultBehaviors. Can you remove the overrides in `initialize` and instead set those as the default behaviors up here? The comments will need to be updated too.

Reply:
```js
// about:accounts display the "Sign in" screen after the users sign in
afterSignInConfirmationPoll: new NavigateBehavior('signin_confirmed'),
// about:accounts display the "Signup complete!" screen after the users verify their email
afterSignUpConfirmationPoll: new NavigateBehavior('signup_confirmed')
```
@shane-tomlinson are these comments fine ?

Reply:
Those comments should be fine.